trussworks/react-uswds
# fix: updates storybook version to fix issues in the a11y addon

# Summary

upgrades storybook version to restore expected behavior from a11y addon automated check behaviors

## Related Issues or PRs

Closes https://github.com/trussworks/react-uswds/issues/2191

## How To Test
```
git checkout main
git pull
yarn install
yarn storybook
```

1. Go to 'localhost:9009'
2. Click on the Accessibility tab at the bottom of the page
3. See Initializing... message is no longer there and it instead reports if the component in the viewport passed automated checks

I have made the necessary updates, and now see the addon functioning as expected locally.

### Screenshots (optional)
<img width="1517" alt="image" src="https://user-images.githubusercontent.com/59394696/191338634-996252c6-4161-413f-aa29-ab6106f312bd.png">
